nginx.conf

```diff
@@ -38,11 +35,7 @@ http {
     server_name _;
 
     location / {
-#       try_files $uri $uri/ =404;
-        try_files $uri $uri/ /index.html;
+    try_files $uri$args $uri$args/ $uri $uri/ /index.html =404;
```

Comment:
args - should be used at the final URI
something like this:
```
location / {
  try_files $uri $uri/ /index.php?$args;
}
```

Pls test your solution.

Reply:
Check for various tutorials online for location and query_params parsing.

Reply:
location / {
  try_files $uri $uri/ /index.php?$args;
}
This solution works

Reply:
perfect!